### DISCUSSION TITLE
Chromium

### DISCUSSION THREAD

**Discussion by BossSay** [adultmm/AdultMediaManager#70] [Show and tell]
I replaced the chrome-win directory (\AdultMediaManager\Bin\Chromium\Win64-901912\chrome-win) with a current stable Win_x64_1081368_chrome-win.zip build. IMHO, scraping performance seems tighter. Subsequently gave the Chromium a settings tweak.

[https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Win_x64/1081368/](url)

**Comment by adultmm** [comment]
Thanks to shared this. 
I cannot upgrade PuppeteerSharp because of some dependencies used by EMM. The current Chrome version is the latest officially tested version with this Puppeteer. 

Which scrapers (sites) are faster with the newer version? Maybe I can force the used browser in Puppeteer and write some tests for it. 

Thanks

**Comment by BossSay** [comment]
Difficult for me to quantify per site. Perhaps it's me being optimistic, however the load times for fanart seem a bit better.
